config/packages/serializer/Centreon/HostConfiguration.Host.yml

```diff
@@ -4,22 +4,35 @@ Centreon\Domain\HostConfiguration\Host:
       type: int
       groups:
         - 'host_conf_min'
-        - 'host_mapper_simulation'
+        - 'mapper_host'
```

Comment:
Can you update the serialization name ? Because `mapper` has no meaning in centreon repository

Reply:
Indeed!